docs/src/main/sphinx/release/release-412.md

```diff
@@ -0,0 +1,47 @@
+# Release 412 (5 Apr 2023)
+
+## General
+
+* Add statistics for optimizer rules to query statistics. The number of rules
+  statistics are collected for can be limited with the
+  `query.reported-rule-stats-limit` configuration property. ({issue}`2578`)
+* Add support for aggregate functions and parameters as arguments for the
+  [`json_object()`](json_object) and [`json_array()`](json_array)
+  functions. ({issue}`16489`, {issue}`16523`, {issue}`16525`)
+* Add {func}`exclude_columns` table function. ({issue}`16584`)
+* Allow table functions to return anonymous columns. ({issue}`16584`)
+* Allow disabling use of the cost-based optimizer for determining partitioning
+  of a stage with the `optimizer.use-cost-based-partitioning` configuration
+  property or the `use_cost_based_partitioning` session property. ({issue}`16781`)
+* Improve performance of queries involving table functions with table arguments. ({issue}`16012`)
+* Improve latency for small queries when fault-tolerant execution is enabled. ({issue}`16103`)
+
+## JDBC driver
+
+* Allow configuring a custom DNS resolver. ({issue}`16647`)
+
+## Delta Lake connector
+
+* Add support for arithmetic binary expressions in check constraints. ({issue}`16721`)
+* Improve performance of queries that only read partition columns. ({issue}`16788`)
+
+## Hive connector
+
+* Fix query failure when bucketing or sorting columns are registered in
+  a metastore in uppercase. ({issue}`16796`)
+* Fix query failure when reading ACID tables with locations containing hidden
+  directories. ({issue}`16773`)
+
+## Iceberg connector
+
+* Fix incorrect results for the `migrate` procedure when the table location
+  contains a hidden directory. ({issue}`16779`)
+
+## Ignite connector
+
+* Add support for `ALTER TABLE ... ADD COLUMN`. ({issue}`16755`)
+
+## Phoenix connector
+
+* Improve performance of queries involving predicates with arithmetic
```

Comment:
what do we normally say for predicate pushdown

Add support for predicate pushdown for predicates with arithmetic expressions 

maybe?

Reply:
Yeah, we usually do that, but I think this is actually a superior explanation to an end user. Probably how we should've been doing it this entire time? I read the proposed release note from @hashhar and personally thought, "yeah, that's an improvement over how we've historically done this."

Reply:
Fair enough .. I am happy to switch to this going forward

Reply:
actually this wording is what Piotr used in the very first complex pushdown release note and I liked it so copied from there.

Reply:
it's more wordy than what we usually use but I think conveys the improvement much better